htslib/khash.h

```diff
@@ -402,12 +435,28 @@ static kh_inline khint_t __ac_X31_hash_string(const char *s)
 	if (h) for (++s ; *s; ++s) h = (h << 5) - h + (khint_t)*s;
 	return h;
 }
+
+/*! @function
+  @abstract     const char* FNV1a hash function
+  @param  s     Pointer to a null terminated string
+  @return       The hash value
+ */
+static kh_inline khint_t __ac_FNV1a_hash_string(const char *s)
+{
+    const khint_t offset_basis = 2166136261;
+    const khint_t FNV_prime = 16777619;
```

Comment:
Can we get consistent indentation here please?  Similarly for __ac_FNV1a_hash_kstring below.  That looks like hard tabs for this file, although there's precedence for 4 spaces in the Wang hash.

I don't mind either way except for not switching half way through a function :-)

Reply:
Spacing has been made consistent.